src/language/mixins.md

```diff
@@ -89,6 +89,50 @@ A `mixin` declaration defines a mixin. A `class` declaration defines a [class][]
 A `mixin class` declaration defines a class that is usable as both a regular class
 and a mixin, with the same name and the same type.
 
+Any restrictions that apply to classes and mixins also apply to mixin classes:
```

Comment:
I'd probably change "classes and mixins" to "classes or mixins" because it's really if there is a restriction for either of those, then the restriction applies to mixin classes to. Otherwise, you could misread this as saying only the restrictions *shared* between mixins and classes are applied to mixin classes.